pkg/identityManager/awsConfig.go

```diff
@@ -16,10 +16,10 @@ package identitymanager
 
 // AwsConfig contains the AWS configuration and access key for the Liqo user and the current EKS cluster.
 type AwsConfig struct {
-	AwsAccessKeyID     string
-	AwsSecretAccessKey string
-	AwsRegion          string
-	AwsClusterName     string
+	AwsAccessKeyID     string `json:"awsAccessKeyID"`
+	AwsSecretAccessKey string `json:"awsSecretAccessKey"`
+	AwsRegion          string `json:"awsRegion"`
+	AwsClusterName     string `json:"awsClusterName"`
```

Comment:
Can we move it inside the new api package? In the future it should cause a **circular dependency**.